### PR TITLE
Wait for postgresql to be started

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: tests/unit/test_cli.py [hunk 1]
```diff
@@ -22,3 +25,30 @@ def test_apply_config_with_plugins(mocker):
...
+    assert engine.connect.called is True
+    assert sleep.called is False
+
+
+def test_check_connectivity_sleep(mocker):
+    sleep = mocker.patch('temboardagent.cli.sleep')
+    from temboardagent.cli import check_connectivity
+
+    engine = mocker.Mock(name='engine')
+    engine.connect.side_effect = Exception()
```
[the remark this answers is not in the document]
Comment (pgiraud) [reply tests/unit/test_cli.py]: Changed. Does it look better?
I chose to use OperationalError to match reality but I don't think it's very important.
Wouldn't it be better to just set a message to a generic Exception and test that raise matches the message?

[the remark this answers is not in the document]
Comment (dlax) [reply tests/unit/test_cli.py]: That's fine now, thanks. Using `Exception` with a specific message would have been fine as well.

[the remark this answers is not in the document]
Comment (pgiraud) [reply tests/unit/test_cli.py]: I changed it again. We save an unrequired import.

**Comment by pgiraud** [comment]
Replaced by #444

**Pull request closed without merging**